contract/wizards/mail_compose_message.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MailComposer(models.TransientModel):
+    _inherit = 'mail.compose.message'
+
+    @api.model
+    def create(self, values):
+        result = super(MailComposer, self).create(values)
+        if self.env.context.get('override_subtype_id'):
+            result.subtype_id = self.env.context.get('override_subtype_id')
```

Comment:
Why do you need to override the subtype?

Reply:
Because it's incompatible define subtype A (_contract-modification_) in `message_post_with_template` function and simulate subtype B (_message_).
It's not possible too force `partner_ids` in `message_post_with_template` function because is redefined according to subtype used in wizard creation.

Reply:
Can we revert then to what we have previously?

Reply:
IMO is not possible to revert, because in "_previous commit_" https://github.com/OCA/contract/commit/25baee81f9998bd2ff7af79d6fe9e505b3939477 fix error to send mail only **contract_modification** followers.

Reply:
OK, let's go for it.